requirements.txt

```diff
@@ -3,7 +3,7 @@ pandas
 scipy
 matplotlib
 h5py
-skorch<0.14
+skorch<0.13
```

Comment:
do we still need this? why? can you add comment here?

Reply:
I reversed @agramfort, but let me explain better why I started restricting the libraries. 

Not really sure why, but there was an API break between 0.13 and 0.14 on skorch. An argument has become mandatory in NeuralNetwork, and the fit method now requires y or something like that.

We noticed with new users, during the code sprint in Basel, that when they tried to install and run the braindecode tests, a lot of errors came up related to this skorch issue.

Another problem is that conda takes longer (+2 hours in some computers) to create a braindecode env. I solved this in CI by switching to Mamba, but installing Mamba on new users' computers takes a lot of work and time. There was one participant who couldn't even install braindecode. This is why I'm tending to restrict the skorch version.